data/transport missions.txt

```diff
@@ -3469,3 +3469,125 @@ mission "Inheritance Redirection"
 				decline
 	on complete
 		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."
+
+
+
+mission "Hauler VI cargo prototype 1"
```

Comment:
Needs a `to offer` property so you don't get it 100% of the time when you visit a neighbor system. Also upping the distance between the source and destination might make sense too. Keeping sushi fresh for one day with refrigeration isn't a very impressive feat. We can do that today. Maybe if the journey took a week, this would make more sense.

Reply:
Changing source to Deep (a planet with fishing villages in its description, so it makes sense to get Sushi there). The mission now only offers on this planet and the player is unlikely to land here often (the planet has no outfitter and shipyards and not many missions go here), so I didn't add other conditions on offering.